image_generation/stable_diffusion_1_5/cpp/scripts/requirements.txt

```diff
@@ -1,8 +1,18 @@
+--extra-index-url https://download.pytorch.org/whl/cpu
+numpy
+openvino>=2023.1.0
+auto-gptq>=0.5.1 # for gptq
+pillow
 torch
-torchvision
-torchaudio
-diffusers
-git+https://github.com/huggingface/optimum-intel.git@cadfab708a87bf677ed52b7e7752a2a5b736fa94
+transformers>=4.33.0
+diffusers>=0.22.0
+optimum>=1.14.0,<1.15.0
+git+https://github.com/huggingface/optimum-intel.git@f248835b16ce4ec054d6d4d629dff4213fe94157
 git+https://github.com/openvinotoolkit/nncf.git
-openvino>=2023.1.0
-onnx
\ No newline at end of file
+packaging
+psutil
+timm
+tiktoken
+onnx
+einops
+transformers_stream_generator
```

Comment:
why do we need all these dependencies? they don't seem to be required for model conversion

Reply:
Thanks, I will update this